catalog-template.yaml

```
@@ -1,1346 +1,3107 @@
+      name: rhacs-3.68
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-3.69
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-3.70
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-3.71
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-3.72
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-3.73
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-3.74
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-4.0
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-4.1
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-4.2
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-4.3
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-4.4
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.channel
+      name: rhacs-4.5
+    message: |
+      This version is no longer supported. Please switch to the `stable` channel or a channel for a version that is still supported.
+  - reference:
+      schema: olm.bundle
+      name: rhacs-operator.v3.62.0
+    message: |
+      This operator bundle version is no longer supported. Please switch to non deprecated bundle version for support.
```

Suggestion, as we also discussed in sync today
```suggestion
      This operator version is no longer supported. Please switch to a newer version that's still in support. Find supported versions in RHACS support policy document: https://access.redhat.com/support/policy/updates/rhacs
```
Same about #forum-acs-docs - it's best to ask docs team to review this message.